script/before_install.sh

```diff
@@ -40,7 +41,7 @@ echo ALPAKA_CI_BOOST_BRANCH_MINOR: "${ALPAKA_CI_BOOST_BRANCH_MINOR}"
 #-------------------------------------------------------------------------------
 # CUDA
 export ALPAKA_CI_INSTALL_CUDA="OFF"
-if [ "${ALPAKA_ACC_GPU_CUDA_ENABLE}" == "ON" ]
+if [[ "${ALPAKA_ACC_GPU_CUDA_ENABLE}" == "ON" && -z "${GITLAB_CI+x}" ]]
```

Comment:
Why do you use the `nvidia/cuda:{version}-devel-ubuntu{version}` container and not the `nvidia/cuda:{version}-base-ubuntu{version}` container and install cuda afterwards, like you suggested for the original alpaka container?

Reply:
discussed offline: using base container and `apt` is more expensive. Container are cached but apt is not cached in our CI.
Never the less the next step will be switching to alpaka container instead of NVIDIA container.